### DISCUSSION TITLE
Feedback: Using local HTTPS

### DISCUSSION THREAD

**Discussion by Cedoriku** [mswjs/mswjs.io#322] [Q&A] [answered]
- [Open page](https://mswjs.io/docs/recipes/using-local-https)

## Feedback
The flags listed are not existing anymore on Chrome. Doc is outdated. Result is I can't use MSW with my local https relied devlopement environment

**Comment by kettanaito** [comment, marked as answer]
Hi, @Cedoriku. Thanks for letting me know. Fixed in 4df6b84.
